### PR TITLE
moving jest to the root of the repo

### PR DESCRIPTION
**Pull request by julien51**
# Description

Since all applications use the jest binary, let's use a single one ;)

# Checklist:

- [X] 1 PR, 1 purpose: my Pull Request applies to a single purpose
  - [X] This PR only contains configuration changes (package.json, etc.)
  - [ ] This PR only contains code changes (if configuration changes are required, do a separate PR first, then re-base)
- [ ] My code follows the style guidelines of this project, including naming conventions
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] If my code adds or changes components, I have written corresponding stories with Storybook
- [ ] I have performed a self-review of my own code
- [ ] If my code involves visual changes, I am adding applicable screenshots to this thread

### CODE REVIEW

**Comment by julien51** [comment]
Also this is the kickoff for the coverage stuff as well ;)

File: unlock-app/jest.config.js [hunk 1]
```diff
@@ -1,13 +1,13 @@
 module.exports = {
   setupFiles: ['<rootDir>/.jest/register-context.js'],
-  setupTestFrameworkScriptFile: '<rootDir>/jest.setup.js',
+  setupFilesAfterEnv: ['<rootDir>/jest.setup.js'],
```
Comment (cellog) [inline unlock-app/jest.config.js]: 👍

File: unlock-app/package.json [hunk 2]
```diff
@@ -19,8 +19,7 @@
     "cross-env": "^5.2.0",
     "ethereumjs-util": "^6.0.0",
     "history": "^4.7.2",
-    "jest": "^23.6.0",
-    "jest-dom": "^3.0.0",
+    "jest-dom": "^3.1.0",
```
Comment (cellog) [inline unlock-app/package.json]: are these jest-dom and terser changes intentional? PR doesn't mention them

Comment (julien51) [reply unlock-app/package.json]: Yes! They are sub-dependencies which were used by jest and that we need to keep.

**Pull request merged**